docs/architecture/grpc-for-wcf-developers/appendix.md

```diff
@@ -6,17 +6,17 @@ ms.date: 09/02/2019
 
 # Appendix A - Transactions
 
-Windows Communication Foundation (WCF) supported distributed transactions, allowing atomic operations to be performed across multiple services. This functionality was based on the [Microsoft Distributed Transaction Coordinator](https://docs.microsoft.com/previous-versions/windows/desktop/ms684146(v=vs.85)).
+Windows Communication Foundation (WCF) supports distributed transactions, allowing you to perform atomic operations across multiple services. This functionality is based on the [Microsoft Distributed Transaction Coordinator](https://docs.microsoft.com/previous-versions/windows/desktop/ms684146(v=vs.85)).
 
-In the modern microservices landscape, this type of automated distributed transaction processing isn't possible. There are too many different technologies at play, including relational databases, NoSQL data stores, and messaging systems, not to mention the mix of operating systems, programming languages and frameworks that may be used in a single environment.
+In the newer, microservices landscape, this type of automated distributed transaction processing isn't possible. There are too many different technologies involved, including relational databases, NoSQL data stores, and messaging systems. There might also be a mix of operating systems, programming languages, and frameworks in use in a single environment.
```

Comment:
I don't feel that the comma fits here.

```suggestion
In the newer microservices landscape, this type of automated distributed transaction processing isn't possible. There are too many different technologies involved, including relational databases, NoSQL data stores, and messaging systems. There might also be a mix of operating systems, programming languages, and frameworks in use in a single environment.
```

Reply:
Without comma, it reads as if there is another microservices landscape, which is older, has been longer on the stage. Thus, the type of processing is possible in that older microservices one.

With comma, the "newer" refers to the "landscape", not to the "microservices landscape". And, the mentioned type of processing is not possible with microservices at all.